linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_all_forwarding/rule.yml

```diff
@@ -18,20 +18,21 @@ identifiers:
     cce@rhel8: CCE-82863-2
     cce@rhel9: CCE-84114-8
     cce@sle15: CCE-85713-6
+    cce@sle12: CCE-83247-7
 
 references:
     cis@rhel7: 3.2.1
     cis@rhel8: 3.1.1
     disa: CCI-000366
     srg: SRG-OS-000480-GPOS-00227
-    nist: CM-7(a),CM-7(b),CM-6(a)
-    nist@sle15: CM-6(b),Cm-6.1(iv)
+    nist: CM-7(a),CM-7(b),CM-6(a),CM-6(b),Cm-6.1(iv)
```

Comment:
Hate to nitpick, but this is the only place in CaC right now that isn't a capitalized CM. :/ 

If you uppercase it, I'll gladly merge :-)